lightning/src/ln/channel.rs

```diff
@@ -936,6 +936,7 @@ pub(crate) struct ShutdownResult {
 	pub(crate) user_channel_id: u128,
 	pub(crate) channel_capacity_satoshis: u64,
 	pub(crate) counterparty_node_id: PublicKey,
+	pub(crate) is_manual_broadcast: bool,
```

Comment:
I though of doing it but I choose the monkey way of doing it by iterating over all the channels, thanks for catching it